handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -62,19 +62,20 @@
             try {
                 String os = normalizeOs(SystemPropertyUtil.get("os.name", ""));
                 String arch = normalizeArch(SystemPropertyUtil.get("os.arch", ""));
-                String libName = "netty-tcnative-" + os + '-' + arch;
-                try {
-                    // First, try loading the platform-specific library. Platform-specific
-                    // libraries will be available if using a tcnative uber jar.
-                    NativeLibraryLoader.load(libName, SSL.class.getClassLoader());
-                } catch (Throwable t) {
-                    // Either the uber jar isn't on classpath, or it doesn't contain a library
-                    // for this platform. Load the default library.
-                    logger.debug("Unable to load platform-specific netty-tcnative library: " +
-                        libName + ". Will attempt to load the default netty-tcnative library.", t);
 
-                    NativeLibraryLoader.load("netty-tcnative", SSL.class.getClassLoader());
+                Set<String> libNames = new LinkedHashSet<String>(3);
+                // First, try loading the platform-specific library. Platform-specific
+                // libraries will be available if using a tcnative uber jar.
+                libNames.add("netty-tcnative-" + os + '-' + arch);
+                if ("linux".equalsIgnoreCase(os)) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "linux" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1192)